osquery/tables/applications/windows/carbon_black.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/tables.h>
+#include <osquery/tables/system/windows/registry.h>
```

Comment:
nit, Use `"osquery/tables/system/windows/registry.h"` for headers not located in the `./include/` directory. This inverts the search path and helps CMake not make mistakes. ;)